corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -14,25 +14,24 @@
 from corehq.apps.userreports.exceptions import UserReportsError
 from corehq.apps.userreports.models import ReportConfiguration
 from corehq.apps.userreports.reports.factory import ReportFactory
+from corehq.apps.app_manager.dbaccessors import get_apps_in_domain
 
 
 class ReportFixturesProvider(object):
     id = 'commcare:reports'
 
-    def __call__(self, user, version, last_sync=None):
+    def __call__(self, user, version, last_sync=None, app=None):
         """
         Generates a report fixture for mobile that can be used by a report module
         """
-        # delay import so that get_apps_in_domain is mockable
-        from corehq.apps.app_manager.dbaccessors import get_apps_in_domain
         if not toggles.MOBILE_UCR.enabled(user.domain):
             return []
 
+        apps = [app] if app else (a for a in get_apps_in_domain(user.domain, include_remote=False))
         report_configs = [
             report_config
-            for app in get_apps_in_domain(user.domain) if isinstance(app, Application)
```

Comment:
@czue, I'm thinking about your feedback "will break if any mobile UCR configs change". Does that mean there's a potential bug here? i.e. Does the app version on the mobile have to be up-to-date otherwise the sync (as it current is in master) could potentially break a report?